docs/features/techdocs/concepts.md

```diff
@@ -8,13 +8,41 @@ Spotify's docs-like-code solution in Backstage
 This page describes concepts that are introduced with Spotify's docs-like-code
 solution in Backstage.
 
-### TechDocs Core Plugin
+### TechDocs Preparer
```

Comment:
I would group the Preparer, Generator, and Publisher under the same heading (as they cover the same concept of building TechDocs). So ## Building Techdocs (or similar) and the 3 concepts under it.